card/lib/cardio.rb

```diff
@@ -50,26 +48,19 @@ def set_config config
   
     def set_paths paths
       @@paths = paths
-      paths.add 'tmp/lib'
-      paths.add 'tmp/set'
-      paths.add 'tmp/set_pattern'
-      paths.add 'db/migrate_deck_cards', :with=>'db/migrate_cards'
+      add_gem_path 'tmp/set', :root => root
+      add_gem_path 'tmp/set_pattern', :root => root
```

Comment:
I added root so that it could handle different gem_roots (at the time there were places where we were referring to different gem roots in at least one of these).

but my point is that there are gem paths (paths to something in the gem dirs) and deck paths (paths to something in the deck dirs).  And using add_gem_path for adding deck paths makes no sense.